azure-pipelines.yml

```diff
@@ -298,30 +298,30 @@ stages:
             continueOnError: true
             condition: always()
 
-        #- job: Linux_FCS
-        #  pool:
-        #    vmImage: ubuntu-16.04
-        #  variables:
-        #  - name: _SignType
-        #    value: Test
-        #  steps:
-        #  - checkout: self
-        #    clean: true
-        #  - script: ./fcs/build.sh Build
-        #    displayName: Build
-
-        - job: MacOS_FCS
+        - job: Linux_FCS
           pool:
-            vmImage: macOS-10.13
+            vmImage: ubuntu-16.04
           variables:
           - name: _SignType
             value: Test
           steps:
           - checkout: self
             clean: true
-          - script: ./fcs/build.sh Build
+          - script: ./fcs/build.sh
```

Comment:
Presumably the `Build` argument was removed here because that's the default target?

Reply:
Yeah, Philip and I talked about it in another thread a bit. I think it would also be reasonable to change this to run tests on every OS instead of just Windows.